Data/Csv/Conversion.hs

```diff
@@ -84,6 +84,9 @@ import qualified Data.ByteString.Lazy as L
 #if MIN_VERSION_bytestring(0,10,4)
 import qualified Data.ByteString.Short as SBS
 #endif
+#if MIN_VERSION_base(4,8,0)
```

Comment:
Please avoid conditional instances; before `base-4.8` `Identity` lived in 'transformers`

Reply:
Would it be okay to add `transformers` as a dependency?

Reply:
@fumieval yes, that would be totally fine, as `transformers` has been a GHC boot library (i.e. bundled with GHC) for years already, and thus bears no significant additional cost.